.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,33 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: "[BUG]"
+labels: bug
+assignees: ''
+
+---
+
+**Describe the bug**
+A clear and concise description of what the bug is.
+
+**To Reproduce**
+Steps to reproduce the behavior:
+1. Go to '...'
+2. Click on '....'
+3. Scroll down to '....'
+4. See error
+
+**Expected behavior**
+A clear and concise description of what you expected to happen.
+
+**Screenshots**
+If applicable, add screenshots to help explain your problem.
+
+**OPTIONAL Real Submitter**
+Who is the real submitter, in case the creation of the issue was done by proxy. This is relevant in case the feature request needs more explanation.
+
+**OPTIONAL Links**
+to additional information (papers, info on algorithms, ...)
+
+**OPTIONAL Priority**
+Priority of this request (Critical, Major, Minor, Trivial; we have labels for blocker and hot)
```

Comment:
Will these labels influence our review process? If not, I would explicitly state this to avoid possible abuse